server.go

```diff
@@ -896,6 +897,11 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB,
 	if _, err := rand.Read(chanIDSeed[:]); err != nil {
 		return nil, err
 	}
+
+	// Initialize the ChainedAcceptor and add a DefaultAcceptor.
+	chainedAcceptor := chanacceptor.NewChainedAcceptor()
+	chainedAcceptor.AddAcceptor(&chanacceptor.DefaultAcceptor{})
```

Comment:
I don't think we need the default acceptor. The chained acceptor can just return true as long as there are no added acceptor.